person.rb

```diff
@@ -24,5 +25,9 @@ def correct_name
     @name
   end
 
+  def add_rentals(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] @iamchristianani great work in your class however in order to achieve many to many relationship between book and person your add rentals method needs a person and date parameters, so that the rental to be created should have three parameters the date, the bok itself and the person renting the book, kindly modify the method.